### ISSUE TITLE
Collaboration Ability

### ISSUE THREAD

**Issue by nblracer880** [hoarder-app/hoarder#627]
### Describe the feature you'd like

Hi, I am the CTO for a smaller MSP. I love using Hoarder personally and for work. Really helps me come back to something cool I see when I don't have time to deep dive it at the moment.

Also helps me quickly find something that was helpful to me for something in the past if I need to reference it again.

I've made some of my techs an account and they like using it as well.

I think it would be really awesome to have some sort of a collaboration feature. I would compare it to a "Collection" in Bitwarden if you are familiar.

### Describe the benefits this would bring to existing Hoarder users

Basically when adding a link or anything else, there would be the option to place it in a shared space where any user can also see / search for things that are put there.

I'm sure this would require quite a bit of work as it probably fundamentally changes a lot of things. But just an idea. Maybe others will also see the benefit to such a functionality.

### Can the goal of this request already be achieved via other means?

Using a shared account would probably accomplish the same thing.

### Have you searched for an existing open/closed issue?

- [X] I have searched for existing issues and none cover my fundamental request

**Comment by MohamedBassem** [comment]
Hey, thanks for describing the usecase. We have plans to support shared lists in #37. Closing this as duplicate for now.
